doc/Extensions/OpenTSDB.md

```diff
@@ -0,0 +1,27 @@
+source: Extensions/OpenTSDB.md
+# Enabling support for OpenTSDB.
+
+This module sends all metrics to OpenTSDB server. You need something like Grafana for graphing.
+
+### Requirements
+ - OpenTSDB
+ - Grafana
+ 
+### What you don't get
+ - Pretty graphs, this is why at present you need Grafana. You need to build your own graphs within Grafana.
+ 
+RRD will continue to function normally so LibreNMS itself should continue to function normally.
+
+configuration in /opt/librenms/includes/defaults.inc.php.
```

Comment:
No, this is the docs. You're telling people config is in includes/defaults.inc.php, users should never be directed to this file and all changes should be in config.php